src/main/java/TaskManager.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+
+public class TaskManager {
+    private final String BOT_NAME = "[iWish]: ";
```

Comment:
Perhaps adding javadoc (brief description of this parameter) according to coding standard